lms/djangoapps/courseware/tests/test_about.py

```diff
@@ -417,6 +419,49 @@ def test_course_price_is_not_visble_in_sidebar(self):
         self.assertNotIn('<span class="important-dates-item-text">$10</span>', resp.content)
 
 
+@attr(shard=1)
+@ddt.ddt
+class AboutSidebarHTMLTestCase(SharedModuleStoreTestCase):
+    """
+    This test case will check the About page for the content in the HTML sidebar.
+    """
+    def setUp(self):
+        super(AboutSidebarHTMLTestCase, self).setUp()
+        self.course = CourseFactory.create()
+
+    @ddt.data(
+        ("", "", False),
+        ("about_sidebar_html", "About Sidebar HTML Heading", False),
+        ("about_sidebar_html", "", False),
+        ("", "", True),
+        ("about_sidebar_html", "About Sidebar HTML Heading", True),
+        ("about_sidebar_html", "", True),
+    )
+    @ddt.unpack
+    def test_html_sidebar_enabled(self, itemfactory_display_name, itemfactory_data, waffle_switch_value):
+        with override_switch(
+            '{}.{}'.format(
+                WAFFLE_NAMESPACE, ENABLE_COURSE_ABOUT_SIDEBAR_HTML
+            ),
+            active=waffle_switch_value
+        ):
+            if itemfactory_display_name:
+                ItemFactory.create(
+                    category="about",
+                    parent_location=self.course.location,
+                    display_name=itemfactory_display_name,
+                    data=itemfactory_data,
+                )
+            url = reverse('about_course', args=[text_type(self.course.id)])
+            resp = self.client.get(url)
+            self.assertEqual(resp.status_code, 200)
+            if waffle_switch_value and itemfactory_display_name and itemfactory_data:
```

Comment:
about sidebar should only be displayed if the waffle switch is enabled AND there is content in studio.